code/model/editableformfields/EditableTextField.php

```diff
@@ -46,6 +49,20 @@ public function getCMSFields()
                     _t('EditableTextField.PLACEHOLDER', 'Placeholder')
                 )
             );
+
+            $fields->addFieldToTab(
+                'Root.Main',
+                DropdownField::create(
+                    'Autocomplete',
+                    _t('EditableTextField.AUTOCOMPLETE', 'Autocomplete'),
+                    $this->config()->get('autocomplete_options'),
+                    _t('EditableTextField.AUTOCOMPLETE', 'Autocomplete')
```

Comment:
This argument is the current value of the dropdown and shouldn't be a translated value, it can probably be removed completely...